docs/microsoft-bookings.md

```diff
@@ -0,0 +1,80 @@
+# How to Configure Microsoft Bookings
+
+This sample takes advantage of the Microsoft 365 Bookings app to power the consumer scheduling experience and create meetings for providers. Follow the steps below to create a Microsoft Bookings calendar and use it in the sample.
+
+### Steps
+
+- [Create a Microsoft Bookings Calendar](#bookingscalendar)
+- [Set the Microsoft Bookings URL in the Sample](#bookingsurl)
+- [Integrate the Sample in Microsoft Bookings](#bookingssampleintegration)
+- [Make an appointment in the Sample](#sampleappointment)
+
+## <a id="bookingscalendar">Create a Microsoft Bookings Calendar</a>
+
+Go to https://outlook.office.com/bookings/homepage to create a Microsoft Bookings calendar.
+
+![bookings-create-calendar.png](../docs/images/bookings/bookings-create-calendar.png)
+
+Ensure the `Make this a Teams meeting` setting is enabled.
+
+![bookings-teams-meeting.png](../docs/images/bookings/bookings-teams-meeting.png)
+
+Ensure the bookings page is publicly accessible.
+
+![bookings-public-page.png](../docs/images/bookings/bookings-public-page.png)
+
+Click `Create calendar`.
+
+## <a id="bookingsurl">Set the Microsoft Bookings URL in the Sample</a>
+
+Copy the Microsoft Bookings URL from your Bookings calendar.
+
+![bookings-page-url.png](../docs/images/bookings/bookings-page-url.png)
+
+### Configure using environment variables
```

Comment:
can we also add here on how to configure in app settings of deployed sample? Although its environment variables, it not very clear where this needs to be set.

Reply:
Let's provide the link to make it more clear https://learn.microsoft.com/en-us/azure/app-service/configure-common?source=recommendations&tabs=portal